### DISCUSSION TITLE
alist利用npm反代后，媒体文件推送到aria2无法下载

### DISCUSSION THREAD

**Discussion by VectorZhao** [AlistGo/alist#4422] [Q&A]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
- [X] I'm sure there are no duplicate issues or discussions.
- [ ] I'm sure it's due to `alist` and not something else(such as `Dependencies` or `Operational`).
- [ ] I'm sure I'm using the latest version

### Alist Version / Alist 版本

v3.17.0

### Driver used / 使用的存储驱动

aliyun，pikpak

### Describe the bug / 问题描述

npm将alist的ip：port进行反代，在alist挂载存储文件出点击下载---aria2下载，aria2一直是下图的状态：
<img width="1353" alt="image" src="https://github.com/alist-org/alist/assets/16644993/b9db5b93-69ec-4eb7-abff-2a4565d360db">
如果用alist的ip：port访问alist的web，然后再发送到aria2下载，就可以下载。
实测，pikpak和阿里云都是上述问题。

### Reproduction / 复现链接

alist

### Logs / 日志

<img width="1353" alt="image" src="https://github.com/alist-org/alist/assets/16644993/23080348-9e80-42f6-9beb-9998bd416141">

**Comment by VectorZhao** [comment]
Releases v3.18.0
问题依然存在

**Comment by AriaLyy** [comment]
+1，同样的问题

**Comment by VectorZhao** [comment]
大佬没修复

**Comment by littleLice** [comment]
+1
